catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -1272,10 +1274,8 @@ private String getFileExtensionForMimeType(String mimeType) {
   private boolean rangeHeaderExists(HttpServletRequest httpRequest) {
     boolean response = false;
 
-    if (null != httpRequest) {
-      if (null != httpRequest.getHeader(HEADER_RANGE)) {
-        response = true;
-      }
+    if (null != httpRequest && null != httpRequest.getHeader(HEADER_RANGE)) {
```

Comment:
Would suggest that we reverse the order of these arguments so that `httpRequest != null && httpRequest.getHeader(HEADER_RANGE) != null`